app/views/posts/index.html.erb

```diff
@@ -1,6 +1,48 @@
-<h1>Here is the post for the user <%= @posts[:id] %></h1>
-<ul>
-<li>Post: <%= @posts[:post_text] %></li>
-<li>Number of likes: <%= @posts[:likes] %></li>
-<li>Number of comments: <%= @posts[:comments] %></li>
-</ul>
\ No newline at end of file
+<div class="container">
+    <div class="row">
+        <div class="col-md-8">
+            <div class="people-nearby">
+                <div class="nearby-user">
+                  <div class="row">
+                    <div class="col-md-2 col-sm-2">
+                      <img src="<%= @user.photo %>" alt="<%= @user.name %>" class="profile-photo-lg">
+                    </div>
+                  <div class="col-md-7 col-sm-7">
+                    <h5 class="mt-2"><%= @user.name %></h5>
+                  </div>
+                  <div class="col-md-3 col-sm-3">
+                    <p class="text-muted mt-5">Number of posts <%= @user.post_counter %></p>
+                  </div>
+                </div>
+              </div>
+            </div>
+    	</div>
+	</div>
+    <div class="card mb-5" style="width: 100%;">
+      <div class="card-body">
+        <h5 class="card-title">Bio</h5>
+        <p class="card-text"><%= @user.bio %></p>
+      </div>
+    </div>
+    <% @posts.each_with_index do |post, index| %>
+    <div class="card mb-2" style="width: 100%;">
+    <div class="card-body">
+      <h5 class="card-title"><%= link_to "Post #{index + 1}", user_post_path(@user, post)%></h5>
+      <p class="card-text"><%= post.text %></p>
+      <div class="col-md-3 text-md-end col-sm-3" style="width: 100%;">
+        <p class="text-muted mt-5"><span>Likes: <%= post.like_counter %></span> <span>Comments: <%= post.comment_counter %></span></p>
+      </div>
+    </div>
+  </div>
+  <div class="card mb-5" style="width: 100%;">
+  <% post.comments.each do |comment| %>
```

Comment:
- [ Optional ] I think it would be nice if you rectify this part of your code to use the `recent_comments` method in the Post model. Currently, it displays all the comments on a given post which might make the page too long especially if every post has many comments. This was supposed to be a project requirement so I highly suggest that you implement this change.

## Project description
| Your design | Application design |
|-----------------|--------------------------|
|![Screenshot from 2023-10-21 15-22-44](https://github.com/mahdinoori2000/blog-app/assets/16392046/fa28eedb-a0c2-45e8-9bf3-7b2b62060565)| ![Screenshot from 2023-10-21 15-25-09](https://github.com/mahdinoori2000/blog-app/assets/16392046/5bd42020-b63e-4b65-9860-11e59e9b05fe)|